pipeline/src/resources/ColorAnnotator.py

```diff
@@ -0,0 +1,59 @@
+
+import tornado.web
+import socket
+import tornado.ioloop
+import json
+
+
+class Annotator(tornado.web.RequestHandler):
+    def __init__(self):
+        self._addr = ''
+        self._port = 12345
+        self._stream = None
+
+    def _initialize(self, sock):
+        client_socket, _ = sock.accept()
+        self._stream = tornado.iostream.IOStream(client_socket)
+        # self._stream.connect((self._addr, self._port))
+
+    def _handle_request(self, sock, fd, events):
+        self._initialize(sock)
+        self._stream.read_until(b'}', callback=self._annotate)
```

Comment:
I don't think this will work with nested JSON. Also consider the possibility that the underlying text has a `"}"` in it.

Reply:
Addressed ("addressed") in #20 - it's ugly, but for now I just have a magic character sequence that Java provides and Python expects.

Reply:
My initial reasoning was that I wasn't sure what the best way to search for the end of JSON was, and I just wanted to be able to get it to work with some basic request. It would (and did!) break with nested JSON. I think Kieran's changes are a much better temporary solution.